include/openssl/bio.h.in

```diff
@@ -209,6 +209,12 @@ extern "C" {
 # define BIO_FLAGS_NONCLEAR_RST  0x400
 # define BIO_FLAGS_IN_EOF        0x800
 
+/* internal BIO:
+ * # define BIO_FLAGS_KTLS_TX_CTRL_MSG 0x1000
+ * # define BIO_FLAGS_KTLS_RX          0x2000
+ * # define BIO_FLAGS_KTLS_TX          0x4000
```

Comment:
I would not repeat the defines here. Just say the values 0x1000 to 0x4000 are reserved for internal KTLS flags.

Please also fix the `BIO_FALGS` typo in the commit message.

Reply:
ok，I  see.